_non-cosmos/ethereum/assetlist.json

```diff
@@ -171,6 +171,49 @@
         }
       ]
     },
+    {
+      "description": "sDAI is a liquid representation of DAI deposited in the Dai savings rate contract.",
+      "type_asset": "erc20",
+      "address": "0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+      "denom_units": [
+        {
+          "denom": "0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+          "exponent": 0,
+          "aliases": [
+            "sdai-wei"
+          ]
+        },
+        {
+          "denom": "sdai",
+          "exponent": 18
+        }
+      ],
+      "base": "0x83F20F44975D03b1b09e64809B757c47f942BEeA",
+      "name": "Savings Dai",
+      "display": "sdai",
+      "symbol": "sDAI",
+      "traces": [
```

Comment:
Based on a quick Google, this is a wrapped or synthetic DAI? I think we should reference DAI in the traces here.

Reply:
It is tokenized version of Dai deposited into Dai savings rate contract on ethereum, so with respect to DAI it is wrapped I guess. But DAI is a synthetic USD stablecoin, so idk if it's more appropriate to say it's wrapped or synthetic?

Reply:
Not 100% familiar with how these terms are used here, apologies if I missed something.

Reply:
Ok. I've made it so that Gravity Bridge sDAI references Ethereum sDAI, and Ethereum sDAI points to Ethereum DAI.